YouTubeSearch/src/redux/actions.ts

```diff
@@ -0,0 +1,14 @@
+import { createAction } from 'redux-actions';
+import {
+  HOME_VIDEOS_REQUEST,
+  SEARCHED_VIDEO_REQUEST,
+  SELECTED_VIDEO_REQUEST,
+  RELATED_VIDEO_REQUEST,
+  VIDEO_DETAILS_REQUEST,
+} from './actionType';
+
+export const homeVideo = createAction(HOME_VIDEOS_REQUEST);

```

Comment:
action из своего названия обозначает действие, поэтому его название должно содержать имя действия, например 
```suggestion
export const fetchHomeVideo = createAction(FETCH_HOME_VIDEOS_REQUEST);

```

Reply:
Хорошей практикой определять тип `payload` у `action`. `redux-action` позволяет это делать - https://thanarie.gitbooks.io/typescript-webpack-book/content/redux-actions.html
 ```suggestion
 interface FetchHomeVideoPayload {
  video: Video;
}

export const fetchHomeVideo = createAction<FetchHomeVideoPayload>(FETCH_HOME_VIDEOS_REQUEST);
```